GLOCKFILE

```diff
@@ -1,5 +1,6 @@
 cmd github.com/cockroachdb/c-protobuf/cmd/protoc
 cmd github.com/cockroachdb/cockroach/protoc-gen-gogoroach
+cmd github.com/cockroachdb/cockroach-prod
```

Comment:
I think you need to do a `glock save` after adding the `cmd` line so that it pins cockroach-prod and its dependencies to a specific revision. But this is risky because glock will not pick the same versions that cockroach-prod has in its own bunchfile. Maybe we should just put the terraform configs in the main repo? The repos were originally split because cockroach-prod had dependencies on things like the aws sdk that we wanted to avoid in the main repo, but now that we're delegating all of that stuff to terraform (and requiring terraform to be installed separately) that's less of an issue.